language-support/java/bindings-rxjava/src/test/scala/com/daml/ledger/rxjava/components/BotTest.scala

```diff
@@ -151,6 +152,19 @@ final class BotTest extends AnyFlatSpec with Matchers with Eventually {
                   commands))
               Single.error(new RuntimeException("expected failure"))
             }
+
```

Comment:
fwiw `rxjava.components` has been deprecated so we _could_ omit multi-party submissions here. That said, given that you’ve already written the code we might as well keep it.